backend/backend/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
somewhat confused as to why there's an `app` dir and a `backend` dir, and why these have repeat configuration